src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -62,11 +67,12 @@ public async Task SynchronizeWithServer_NotInConnectedMode_StoreCleared()
 
             configurationProvider.Setup(x => x.GetConfiguration()).Returns(BindingConfiguration.Standalone);
 
-            var testSubject = CreateTestSubject(
```

Comment:
Miscellaneous change: there were previously two `CreateTestSubject` overloads, which I've modified to support the new tests. That made it simpler to change this existing test just to call the constructor directly.